### PR TITLE
CU-86a265zxk - NEON3 - Settings Screen - Implement Global Network Con…

### PR DESCRIPTION
**Pull request by yumiuehara**
…figuration

**What current issue(s) from Trello/Github does this address?**

**What problem does this PR solve?**

**How did you solve this problem?**

**How did you make sure your solution works?**

**Are there any special changes in the code that we should be aware of?**

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-86a265zxk NEON3 - Settings Screen - Implement Global Network Configuration](https://app.clickup.com/t/86a265zxk)

File: src/renderer/src/components/Sidebar/index.tsx [hunk 1]
```diff
@@ -29,8 +30,8 @@ export const Sidebar = (): JSX.Element => {
 
   return (
     <aside className="bg-gray-800 h-screen w-[4rem] min-w-[4rem] flex flex-col">
-      <div className="flex justify-center py-4">
-        <NeonLogoIcon className="fill-neon" />
+      <div className={StyleHelper.mergeStyles('flex justify-center py-4')}>
```
Comment (raulduartep) [inline src/renderer/src/components/Sidebar/index.tsx]: StyleHelper.mergeStyles is not necessary

File: src/renderer/src/components/RadioGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+type ItemProps = RadixRadioGroup.RadioGroupItemProps & { label: string; leftIcon?: JSX.Element }
+
+const Root = (props: RadixRadioGroup.RadioGroupProps) => {
+  return <RadixRadioGroup.Root {...props} />
+}
+
+const Item = ({ label, leftIcon, className, ...props }: ItemProps) => {
+  return (
+    <div className="flex flex-row gap-y-2 gap-x-4 h-[40px] items-center justify-between hover:bg-asphalt border-b border-gray-300/30">
```
Comment (raulduartep) [inline src/renderer/src/components/RadioGroup.tsx]: Use rem instead of px

File: src/renderer/src/components/RadioGroup.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <div className="flex items-center">
+        {leftIcon && leftIcon}
+        <label>{label}</label>
+      </div>
+
+      <RadixRadioGroup.Item
+        className={StyleHelper.mergeStyles(
+          'data-[state=unchecked]:border-gray-300 data-[state=checked]:border-neon border border-2 bg-transparent w-[16px] h-[16px] rounded-full outline-none cursor-pointer mx-4',
```
Comment (raulduartep) [inline src/renderer/src/components/RadioGroup.tsx]: Use rem instead of px

File: src/renderer/src/components/RadioGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+          className
+        )}
+        {...props}
+      >
+        <RadixRadioGroup.Indicator className="flex items-center justify-center w-full h-full relative after:content-[''] after:block after:w-[8px] after:h-[8px] after:rounded-[50%] after:bg-neon" />
```
Comment (raulduartep) [inline src/renderer/src/components/RadioGroup.tsx]: Use rem instead of px

File: src/renderer/src/components/TestnetBanner.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+import { useNetworkTypeSelector } from '@renderer/hooks/useSettingsSelector'
+
+const Banner = () => {
+  const { t: tCommon } = useTranslation('common', { keyPrefix: 'networkTypeLabel' })
+  const { networkType } = useNetworkTypeSelector()
+
+  if (networkType !== 'testnet') return <></>
+  return (
+    <div className="w-screen border-t-3 border-purple flex justify-center absolute top-0 left-0 z-50">
+      <div className="w-[18px] h-[18px] overflow-hidden relative before:content-[''] before:block before:w-[100%] before:h-[100%] before:absolute before:top-0 before:left-0 before:shadow-[9px_-9px_0_0] before:shadow-purple before:rounded-[50%]" />
```
Comment (raulduartep) [inline src/renderer/src/components/TestnetBanner.tsx]: Use rem instead of px

File: src/renderer/src/components/TestnetBanner.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      <span className="text-white text-[10px] font-medium px-2.5 pb-1 pt-0.5 tracking-wide rounded-b-md bg-purple uppercase">
```
Comment (raulduartep) [inline src/renderer/src/components/TestnetBanner.tsx]: Use text-1xs instead of text-[10px]

File: src/renderer/src/components/TestnetBanner.tsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        {tCommon('testnet')}
+      </span>
+      <div className="w-[18px] h-[18px] overflow-hidden relative before:content-[''] before:block before:w-[100%] before:h-[100%] before:absolute before:top-0 before:left-0 before:shadow-[-9px_-9px_0_0] before:shadow-purple before:rounded-[50%]" />
```
Comment (raulduartep) [inline src/renderer/src/components/TestnetBanner.tsx]: Use rem instead of px

File: src/renderer/src/routes/modals/NetworkSelection/index.tsx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+      <div className="flex flex-col h-full justify-between">
+        <div>
+          <div className="mb-5">{t('selectNetwork')}</div>
+          <RadioGroup.Root value={selectedNetwork} onValueChange={onSelectRadioItem}>
+            {AVAILABLE_NETWORKS.map(network => (
+              <RadioGroup.Item
+                label={commonT(network)}
+                value={network}
+                key={network}
+                leftIcon={<div className={`w-[6.5px] h-[6.5px] rounded mx-4 ${COLOR_BY_NETWORK[network]}`} />}
```
Comment (raulduartep) [inline src/renderer/src/routes/modals/NetworkSelection/index.tsx]: Use rem instead of px

**Pull request merged**